presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -114,6 +114,7 @@
     private int fileSystemMaxCacheSize = 1000;
 
     private boolean writesToNonManagedTablesEnabled;
+    private boolean tableStatisticsEnabled;
```

Comment:
Please keep refactor changes and semantic changes in separate commits (if not separate PR). As you noticed refactor changes seem to be  ok to be merged, while semantic changes are questionable. 

I think refactor changes would be enough to handle your case, you would need to just update the configuration.